### PR TITLE
breaking: Correct video composition date and callback types

### PR DESCRIPTION
**Pull request by childish-sambino**

### CODE REVIEW

File: src/main/java/com/twilio/rest/video/v1/Composition.java [hunk 1]
```diff
@@ -234,8 +234,8 @@ private Composition(@JsonProperty("account_sid")
         this.accountSid = accountSid;
         this.status = status;
         this.dateCreated = DateConverter.iso8601DateTimeFromString(dateCreated);
-        this.dateCompleted = dateCompleted;
-        this.dateDeleted = dateDeleted;
+        this.dateCompleted = DateConverter.iso8601DateTimeFromString(dateCompleted);
+        this.dateDeleted = DateConverter.iso8601DateTimeFromString(dateDeleted);
```
Comment (eshanholtz) [inline src/main/java/com/twilio/rest/video/v1/Composition.java]: Need matching DateConverter.dateStringFromDateTime serializer function (only have DateConverter.dateStringFromLocalDate)

Comment (childish-sambino) [reply src/main/java/com/twilio/rest/video/v1/Composition.java]: The type on the input parameter changed when it shouldn't have. Working on a fix.

File: src/main/java/com/twilio/rest/video/v1/Composition.java [hunk 2]
```diff
@@ -202,9 +202,9 @@ private Composition(@JsonProperty("account_sid")
                         @JsonProperty("date_created")
                         final String dateCreated,
```
Comment (eshanholtz) [inline src/main/java/com/twilio/rest/video/v1/Composition.java]: Why is this a String if the others are DateTime objects?

Comment (childish-sambino) [reply src/main/java/com/twilio/rest/video/v1/Composition.java]: See other comment.

Comment (eshanholtz) [reply src/main/java/com/twilio/rest/video/v1/Composition.java]: 👍

File: src/main/java/com/twilio/rest/video/v1/Composition.java (hunk 1 above)
Comment (childish-sambino) [reply src/main/java/com/twilio/rest/video/v1/Composition.java]: Updated.

**Pull request merged**